INZFS.MVC/Views/FundApplication/MultiRadioInput.cshtml

```diff
@@ -0,0 +1,95 @@
+
+@model MultiRadioInputModel
+@using INZFS.MVC.Models.DynamicForm
+@using INZFS.MVC.TagHelpers
+
+@if (!string.IsNullOrEmpty(Model.PreviousPageName))
+{
+    <a class="govuk-back-link" id="BackButton" asp-area="INZFS.MVC" asp-controller="FundApplication" asp-action="section" asp-route-pagename="@Model.PreviousPageName">
+        Back to previous question
+    </a>
+}
+else
+{
+<a class="govuk-back-link" id="BackButton" asp-area="INZFS.MVC" asp-controller="FundApplication" asp-action="section" asp-route-pagename="@Model.SectionInfo.Url">
+    @Model.SectionInfo.Title
+</a>
+}
+    <span class="govuk-caption-l">Question @Model.QuestionNumber of @Model.TotalQuestions</span>
+
+<form asp-action="save" asp-route-pagename="@Model.PageName" method="post" enctype="multipart/form-data">
+
+    <input type="hidden" name="PageName" value="@Model.PageName" />
+    <input type="hidden" name="Kind" value="@nameof(MultiRadioInputModel)" />
+
+    @if (!ViewData.ModelState.IsValid)
```

Comment:
Please use newly created partial view here. Refer to the other views for examples

Reply:
Done